les/api.go

```diff
@@ -352,3 +389,38 @@ func (api *PrivateLightAPI) GetCheckpointContractAddress() (string, error) {
 	}
 	return api.backend.oracle.config.Address.Hex(), nil
 }
+
+//API should be for light clients of les protocol
+type LightClientAPI struct {
+	le *LightEthereum
+}
+
+func NewLightClientAPI(le *LightEthereum) *LightClientAPI {
+	return &LightClientAPI{le}
+}
+
+func (api *LightClientAPI) GatewayFeeCache() map[string]*GatewayFeeInformation {
+	return api.le.handler.gatewayFeeCache.gatewayFeeMap
+}
+
+// RequestPeerGatewayFees updates cache by pulling gateway fee peer nodes
+func (api *LightClientAPI) RequestPeerGatewayFees() error {
+	peerNodes := api.le.peers.AllPeers()
+	for _, peerNode := range peerNodes {
+		cost := peerNode.GetRequestCost(GetGatewayFeeMsg, int(1))
+		err := peerNode.RequestGatewayFee(genReqID(), cost)
+		if err != nil {
+			continue
```

Comment:
This error should not be ignored. it should at least be logged to the Geth console. Also, as written this `continue` can be omitted because this is the last statement of the loop.